gr-dtv/lib/atsc/atsc_deinterleaver_impl.cc

```diff
@@ -30,25 +30,22 @@ atsc_deinterleaver_impl::atsc_deinterleaver_impl()
                      io_signature::make(1, 1, sizeof(atsc_mpeg_packet_rs_encoded))),
       alignment_fifo(156)
 {
-    m_fifo.resize(52);
+    m_fifo.reserve(s_interleavers);
 
-    for (int i = 0; i < 52; i++)
-        m_fifo[52 - 1 - i] = new interleaver_fifo<unsigned char>(i * 4);
+    for (int i = 0; i < s_interleavers; i++)
+        m_fifo.emplace_back((52 - 1 - i) * 4);
```

Comment:
Is `s_interleaver` a constant? aka `s_interleaver = 52;`? In that case, I'd suggest to use the constant consistently, I assume it would be correct to replace `52` with `s_interleaver` as well. Also, the other literals might be good candidates for replacement too. Well, only `4`.

Found it: `static constexpr int s_interleavers = 52;`. Since this is a constant, would it be better to spell it `S_INTERLEAVERS`?

Reply:
Static member variables [should be prefixed with `s_`](https://github.com/gnuradio/greps/blob/master/grep-0001-coding-guidelines.md#code-formatting-guidelines).

I could make it a global variable in `gr::dtv` and capitalize it, but there's no existing namespace outside the class for just atsc.